src/clients/twitch.ts

```diff
@@ -83,6 +84,38 @@ function registerTwitch(): void
             if(!bridge.targetDiscordChannel)
                 throw new Error('Cannot find Discord channel.');
 
+            /**
+                    'reply-parent-display-name': 'testingaccount__',
+                    'reply-parent-msg-body': 'test',
+                    'reply-parent-msg-id': 'f279b175-7100-4486-bb96-c188ea102bbd',
+                    'reply-parent-user-id': '821973125',
+  '                 'reply-parent-user-login': 'testingaccount__',
+
+                    Those keys will exist on the userState.tags (Map<String, String>) map, if it
+                    is a reply.
+            */
+
+            // if there is a reply parent display name
+            // we know it's a reply.
+            if(userState.tags.get('reply-parent-display-name'))
+            {
+                // get the linked list node from the Twitch ID that was replied to
+                const fetchedNode: linkedListNode = bridge.discordTwitchCacheMap.get(userState.tags.get('reply-parent-msg-id'));
+
+                if(!fetchedNode)
+                    return;
+
+                try
+                {
```

Comment:
I wish javascript could have single-liner try catches  - of all the items that require curly braces... this one's the outlier besides objects themselves XP

Reply:
Same here, although I don't understand the purpose of this comment; not something I can control within this PR.